src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/validation/singleField/singleFieldValidationActions.ts

```diff
@@ -14,13 +12,21 @@ export interface IRunSingleFieldValidationActionFulfilled extends Action {
   validations: any;
 }
 
-export function runSingleFieldValidationAction(
-  url: string,
-  dataModelBinding?: string,
-): IRunSingleFieldValidationAction {
+export interface ISetCurrentDataModelBindingAction extends Action {
+  dataModelBinding?: string;
+}
+
+export function runSingleFieldValidationAction(): IRunSingleFieldValidationAction {
   return {
     type: ActionTypes.RUN_SINGLE_FIELD_VALIDATION,
-    url,
+  };
+}
+
+export function setCurrentDataModelBindingAction(
```

Comment:
Seeing this in context with everything else, I think we should re-name this something like `setCurrentSingleFieldValidation` for consistency (since we use `currentSingleFieldValidation` in the reducer).